code/linear_model.py

```diff
@@ -0,0 +1,156 @@
+import project_config
+import numpy as np
+import numpy.linalg as npl
+import matplotlib.pyplot as plt
+import nibabel as nib
+from general_utils import form_cond_filepath, prepare_data_single, prepare_mask
+from stimuli_revised import events2neural_std
+from conv import conv_target_non_target, conv_std
+from gaussian_filter import spatial_smooth
+from matplotlib import colors
+from hypothesis import perform_t_tests
+
+import pdb
+
+def single_subject_linear_model(standard_source_prefix, cond_filepath_prefix, subject_num, task_num):
+
+  data = prepare_data_single(subject_num, task_num, True, standard_source_prefix)
+
+  n_trs = data.shape[-1] + 5
+
+  cond_filename_003 = form_cond_filepath(subject_num, task_num, "003", cond_filepath_prefix)
+  cond_filename_005 = form_cond_filepath(subject_num, task_num, "005", cond_filepath_prefix)
+  cond_filename_001 = form_cond_filepath(subject_num, task_num, "001", cond_filepath_prefix)
+  cond_filename_004 = form_cond_filepath(subject_num, task_num, "004", cond_filepath_prefix)
+
+  # TODO: put cond007 back to 003
+  cond_filename_007 = form_cond_filepath(subject_num, task_num, "007", cond_filepath_prefix)
```

Comment:
I plan to put errors into a separate regressor. This is since we don't know what errors they are (whether the subject misses a target or mistakenly report a non-target). 

Did you find anywhere in the report that says the error is always a miss for target? It could be a false positive for a non-target too right?